enhancements/two-node-fencing/tnf.md

```diff
@@ -561,54 +610,49 @@ This proposal is an alternative architecture to Single-node and MicroShift, so i
-   that the operator could later be used to communicate information about pacemaker to a cluster admin in case the resource and/or fencing agents become unhealthy.
+   We've talked over the pros and cons of a new operator to handle aspects of the TNF setup. The primary job of a TNF setup operator would be to initialize pacemaker and to ensure that it reaches a
+   healthy state. This becomes a simple way of kicking off the transition from CEO controlled etcd to RHEL-HA controlled etcd. As an operator, it can also degrade during installation to ensure that
+   installation fails if fencing credentials are invalid or the etcd containers cannot be started. The last benefit is that the operator could later be used to communicate information about pacemaker
+   to a cluster admin in case the resource and/or fencing agents become unhealthy.
 
-   After some discussion, we're prioritizing an exploration of a solution to this initialization without introducing a new operator. The operator that is closest in scope
-   to pacemaker initialization is the cluster-etcd-operator. Ideally, we could have it be responsible for kicking off the initialization of pacemaker, since the core of a
-   successful TNF setup is to ensure etcd ownership is transitioned to a healthy RHEL-HA deployment. While it is a little unorthodox for a core operator to initialize an external
-   component, that component is tightly coupled with the health of etcd to begin with and they benefit from being deployed and tested together.  Additionally, most cases that
-   would result in pacemaker failing to initialize would result in CEO being degraded as well. One concern raised for this approach is that we may introduce a greater security
-   risk since CEO permissions need to be elevated so that a container can run as root to initialize pacemaker. The other challenge to solve with this approach is how we
-   communicate problems discovered by pacemaker to the user.
+   After some discussion, we're prioritizing an exploration of a solution to this initialization without introducing a new operator. The operator that is closest in scope to pacemaker initialization
+   is the cluster-etcd-operator. Ideally, we could have it be responsible for kicking off the initialization of pacemaker, since the core of a successful TNF setup is to ensure etcd ownership is
+   transitioned to a healthy RHEL-HA deployment. While it is a little unorthodox for a core operator to initialize an external component, that component is tightly coupled with the health of etcd to
+   begin with and they benefit from being deployed and tested together.  Additionally, most cases that would result in pacemaker failing to initialize would result in CEO being degraded as well. One
+   concern raised for this approach is that we may introduce a greater security risk since CEO permissions need to be elevated so that a container can run as root to initialize pacemaker. The other
```

Comment:
> CEO permissions need to be elevated so that a container can run as root

I'm still investigating this, it's more complicated than root or non-root, there are multiple security related settings in a pod / container definition

Reply:
I expect this to be updated to address these details as part of https://issues.redhat.com/browse/OCPEDGE-1461. My goal for this section was to mostly just get something in the section that roughly maps to our current understanding.

Reply:
As above, I didn't actually change this section. I just added a new commit which has my editor wrap the text at 200 characters so we could avoid having manually wrapped sections and unwrapped sections.

200 characters was chosen because that's the shortest width I could make the widest table.